Loop/Managers/Service.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Service.swift
+//  Loop
+//
+//  Created by Darin Krauss on 5/17/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+import MockKit
+import AmplitudeServiceKit
+import LogglyServiceKit
+import NightscoutServiceKit
+
+
+/// The order here specifies the order in the service selection popup
+let serviceTypes: [Service.Type] = [
+    NightscoutService.self,
+    LogglyService.self,
+    AmplitudeService.self,
+    MockService.self,
```

Comment:
Should the MockService be limited to just debug builds?

Reply:
Yes. Will make that change.